src/lib/types.ts

```diff
@@ -23,6 +25,11 @@ export type SupabaseClientOptions = {
    * A storage provider. Used to store the logged in session.
    */
   localStorage?: Storage
+
+  /**
+   * Options passed to the realtime-js instance
+   */
+  realtimeOptions?: RealtimeClientOptions
```

Comment:
@quicksnap based off earlier comment recommending change to `realtime`:

```suggestion
  realtime?: RealtimeClientOptions
```

Reply:
updated!